ush/python/pygfs/task/marine_bmat.py

```diff
@@ -234,6 +235,8 @@ def finalize(self: Task) -> None:
             src = os.path.join(self.task_config.DATAstaticb, f"{diff_type}_ocean.nc")
             dest = os.path.join(self.task_config.COMOUT_OCEAN_BMATRIX,
                                 f"{APREFIX}{diff_type}_ocean.nc")
```

Comment:
```suggestion
                                f"{APREFIX_OCEAN}{diff_type}.nc")
```
I think a cleaner way than this would be to declare
`APREFIX_OCEAN` and `APREFIX_ICE` in the `__init__`  `localdict` as:
```
'APREFIX_OCEAN': f"{self.task_config.RUN}.ocean.t{self.task_config.cyc:02d}z.",
'APREFIX_ICE': f"{self.task_config.RUN}.ice.t{self.task_config.cyc:02d}z.",
```
and then use them as needed where appropriate.
This makes it cleaner and explicit instead of a method `rename_file`

Reply:
Done